python/cudf/cudf/pylibcudf_tests/common/utils.py

```diff
@@ -24,28 +24,28 @@ def metadata_from_arrow_array(
     return metadata
 
 
-def assert_column_eq(plc_column: plc.Column, pa_array: pa.Array) -> None:
+def assert_column_eq(expect: pa.Array, got: plc.Column) -> None:
```

Comment:
Do we want to model this on https://github.com/rapidsai/cudf/blob/6d5f9653debe57c7eb52f42fb980d38451a9a460/python/cudf/cudf/testing/_utils.py#L116?

We may not always be comparing an “expected/actual” result. The assert_eq just uses left/right names. The order is perhaps better defined by the caller, and hint at that intended use case in a docstring?

Additionally, it’s not obvious that this function expects a PyArrow and a pylibcudf object. Should we normalize this to accept either argument as either type, more like assert_eq?

Reply:
I updated this to be more symmetric, although I think we're stuck with having to have at least _one_ of the objects be a pyarrow array, even if we don't know which one, unless we want to make broader changes. This is due to needing the metadata from the pyarrow array to compare nested data.